app/views/authorised_to_view/authorised_to_view_search.scala.html

```diff
@@ -54,7 +56,9 @@
             classes = Some("govuk-!-width-two-thirds"),
             labelClasses = Some("govuk-visually-hidden")
         )
+
         @button(messages("cf.search.authorities.button"), None)
+        <br>
```

Comment:
I would probably use the spacing classes from the style guide https://design-system.service.gov.uk/styles/spacing/